lib-database-entities/src/commonMain/kotlin/com/ustadmobile/lib/db/entities/PersonWithStatementDisplay.kt

```diff
@@ -0,0 +1,26 @@
+package com.ustadmobile.lib.db.entities
```

Comment:
This isn't really with "statementdisplay". That implies that it is linked to a specific statement, which it isn't. I would call this PersonWithAttemptsSummary